src/main/java/fr/socrates/domain/candidate/CandidateServiceImpl.java

```diff
@@ -54,18 +54,17 @@ private void saveCandidate(Candidate candidate) throws CandidatePersisteDataExce
 
 
     @Override
-    public void update(EMail email, AccommodationChoices accommodationChoices, ContactInformation contactInformation) {
```

Comment:
Why two updates when we can do ONE update of the candidate ?

Reply:
talked about this with Boa. Germain confirmed that it is better to update the part of the candidate object concerned, and not the total object.